api/src/graphql/typeDefs.js

```diff
@@ -2,14 +2,21 @@ const typeDefs = `
   schema {
     query: Query
     mutation: Mutation
+    subscription: Subscription
+  }
+
+  type Subscription {
+    realityCreated: Reality
+    realityDeleted: Reality
+    realityUpdated: Reality
   }
 
   type Query {
     persons(search: String): [Person]
     person(email: String!): Person
     needs(search: String): [Need]
     need(nodeId: ID!): Need
-    responsibilities(search: String): [Responsibility]
+    responsibilities(nodeId: ID, search: String): [Responsibility]
```

Comment:
See comments above about adding this nodeId argument

Reply:
Fixed